lib/rspec/hive/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module RSpec
   module Hive
-    VERSION = '0.6.2'
+    VERSION = '0.6.3'
```

Comment:
didn't we bump it higher in other PR?

Reply:
for some testing purposes it got uploaded as 0.7.0 to private rubygems repo but it's not longer valid and it was not in another PR